core/server/master/src/main/java/alluxio/master/file/meta/InodeTreePersistentState.java

```diff
@@ -687,6 +753,11 @@ private long getIdFromPath(Path path) {
 
   @Override
   public boolean processJournalEntry(JournalEntry entry) {
+    // Account for entry in retry-cache.
+    if (entry.hasOperationId()) {
```

Comment:
Will it be better to cache after the entry is applied?

Reply:
Right.